### DISCUSSION TITLE
Change how beatmap sorting methods appear in the sorting method quick select

### DISCUSSION THREAD

**Discussion by jal-ei** [ppy/osu#29623] [Ideas]
I personally frequently swap between sorting my beatmaps by "Date Added" and "Title", but "Title" is always removed from quick select, since quick select picks the first 6 sorting methods from the alphabetical list of sorting methods to display.
My idea is that instead quick select should display the 6 last used sorting methods. It'll be far more for convenient for lots of users, and makes more sense.
For the sake of clarity, this is what I mean by quick select:
![image](https://github.com/user-attachments/assets/3038dc07-d9f4-4f7f-814c-e7cc7f7383c1)

**Comment by peppy** [comment]
Recent used filters should be remembered between startups.

**Reply by jal-ei** [reply]
Hey sorry, do you mean that it should already be happening?
